modules/agent-installer-architectures.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent-based-installer/preparing-to-install-with-agent-based-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="agent-install-verifying-architectures_{context}"]
+= Verifying the supported architecture for installing an Agent-based Installer cluster
+
+Before installing an {product-title} cluster using the Agent-based Installer, you can verify the supported architecture on which you can install the cluster. This procedure is optional.
+
+.Prerequisites: 
+
+* You installed the {oc-first}.
+* You have downloaded the installation program.
+
+.Procedure: 
+
+. Log in to the {oc-first}.
+
+. Check your release payload by running the following command: 
+[source,terminal]
++
+----
+$ ./openshift-install version
+----
++
+.Example output 
+[source,terminal]
+----
+./openshift-install 4.16.0
+built from commit abc123def456
+release image quay.io/openshift-release-dev/ocp-release@sha256:123abc456def789ghi012jkl345mno678pqr901stu234vwx567yz0
+release architecture amd64
+----
++
+If you are using the release image with the `multi` payload, the `release architecture` displayed in the output of this command is the default architecture.
+
+. To check the architecture of the payload, run the following command:
+[source,terminal]
++
+----
+$ oc adm release info <release-image> -o jsonpath="{ .metadata.metadata}" <1>
```

Comment:
User-replaced variables should be underscores: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#command-syntax-for-replaceable-values

```suggestion
$ oc adm release info <release_image> -o jsonpath="{ .metadata.metadata}" <1>
```

Reply:
Updated as per the suggestion.